odoo/modules/neutralize.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import textwrap
+
+import odoo
+from .module import get_manifest
+
+
+def get_installed_modules(cursor):
+    cursor.execute('''
+        SELECT name FROM ir_module_module
+        WHERE state IN ('installed', 'to upgrade', 'to remove', 'to install');
+    ''')
+    return [result[0] for result in cursor.fetchall()]
+
+def get_neutralization_sql(modules):
+    queries = []
+
+    # neutralization for each module
+    for module in modules:
+        manifest = get_manifest(module)
+        if manifest and manifest['neutralize']:
+            f = odoo.modules.get_module_resource(module, manifest['neutralize'])
+            with odoo.tools.misc.file_open(f) as neutralize_sql:
+                queries.append(neutralize_sql.read())
+
+    # neutralization flags for the database
+    queries.append(textwrap.dedent('''INSERT INTO ir_config_parameter(key, value)
+        VALUES ('database.is_neutralized', true)
+        ON CONFLICT (key)
+        DO UPDATE SET value = true;
+    '''))
```

Comment:
Well, `textwrap.dedent` is useless when used that way: it removes the indentation that is common to all lines, and your string has none
```suggestion
    queries.append(textwrap.dedent('''\
        INSERT INTO ir_config_parameter(key, value)
        VALUES ('database.is_neutralized', true)
        ON CONFLICT (key)
        DO UPDATE SET value = true;
    '''))
```